### PR TITLE
Scaffolding: security, contributions, and code of conduct

### PR DESCRIPTION
**Pull request by chayim**
Adding guidelines for responsible disclosure, and contributions, as part of general open source work. These are loosely patterned after things I've done in the past.

The code of conduct comes from [the contributor covenant](https://www.contributor-covenant.org/version/2/1/code_of_conduct/code_of_conduct.txt). The contributing guidelines are influenced by other past projects such as [this one](https://github.com/redis/redis-py/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+Scope
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+Enforcement
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+this email address: redis@redis.io.
```
Comment (scottr-uai) [inline CODE_OF_CONDUCT.md]: redis email

File: SECURTIY.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Reporting a vulnerability
+
+## Introduction
```
Comment (scottr-uai) [inline SECURTIY.md]: This files name has a type, `SECURTIY.md` -> `SECURITY.md`

**Review by scottr-uai — DISMISSED** [review]
Couple of typos is all, thanks for doing this, will be valuable to have this codified.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (chayim) [reply CODE_OF_CONDUCT.md]: Thank you. I took the code of conduct from there, and neglected to update the mail.

**Review by scottr-uai — APPROVED** [review]

**Pull request merged**